src/index.js

```diff
@@ -1,12 +1,24 @@
-import Vue from 'vue'
 import VuePlyr from './VuePlyr.vue'
 
 const Components = {
-  VuePlyr
+    VuePlyr
 }
 
-Object.keys(Components).forEach(component => {
-  Vue.component(Components[component].name, Components[component])
-})
+const VuePlyrPlugin = {
+    install(Vue, options) {
+        Object.keys(Components).forEach(component => {
+            Vue.component(Components[component].name, Components[component])
+        })
+    }
+}
+
+if (typeof window !== 'undefined' && window.Vue) {
```

Comment:
This part actually doesn't work, because Vue isn't declared in the `window` scope unless you install it using something like unpkg. Using npm as normal leaves the `window` variable untouched. Remove this block please.